docs/source/crawler/basics.rst

```diff
@@ -48,19 +48,23 @@ would need to be provided with specific input. The simplest pipeline could look
 ...     annex
 ...     ]
 
-which crawls in a website and is provided as input to the first function. `a_href_match`
-then works to output all files that end in `.mat`, and those files are lastly inputted to
-`annex` which simply annexes them.
+in which the first generator (method of a class) is provided an input and crawls a website.
+`a_href_match` then works to output all files that end in `.mat`, and those files are
+lastly inputted to `annex` which simply annexes them.
 
 .. note::
     Since pipelines depend heavily on generators, these generators must yield in order
     for an output to be produced. If a generator fails to yield, then the pipeline
-    can no longer continue and it stopped at that generator.
+    can no longer continue and it is stopped at that generator.
 
 Subpipelines
 ------------
 
 A subpipline is a pipeline that lives within a greater pipeline and is also denoted by `[]`.
 Two subpipelines that exist on top of one another will take in the same input, but process it
 with different generators. This functionality allows for the same input to be handled in two
-or more (depending on the number of subpipelines) different manners. 
\ No newline at end of file
+or more (depending on the number of subpipelines) different manners.
+
+TODO: 'FinishPipeline` exception here
+`FinishPipeline <http://docs.datalad.org/en/latest/generated/datalad.crawler.pipeline.html
+#datalad.crawler.pipeline.FinishPipeline>`_
```

Comment:
@yarikoptic is this accurate for using sphinx to reference to the API, so it would make a hyperlink?